packages/yarnpkg-core/sources/Configuration.ts

```diff
@@ -244,6 +244,11 @@ export const coreDefinitions: {[coreSettingName: string]: SettingsDefinition} =
     type: SettingsType.BOOLEAN,
     default: true,
   },
+  yarn1MutexFilename: {
+    description: `When repositories that use yarn1 are included as dependencies, this file is used as a mutex to avoid concurrency issues.`,
+    type: SettingsType.ABSOLUTE_PATH,
+    default: `./.yarn/.yarn-mutex`,
+  },
```

Comment:
If we're going to set the mutex for Yarn Classic I don't see the point of making it configurable.

Reply:
I just wasn't sure this would always be a writable path. If someone has a locked down user in a docker image with least privilege, it seemed possible they might not have write access to `.yarn`.
I was just trying to make it flexible. I had also noticed that the global and cache paths are configurable, so thought maybe all the `.yarn/*` paths were... though looking at it again, it seems stuff like `.yarn/releases`, `/sdks`, `/plugins` etc are not configurable, so maybe yarn just doesn't really work if `.yarn` isn't a writable path.
I'd be fine removing this config option, but just wasn't sure how configurable to keep this 🤷